moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h

```diff
@@ -165,18 +165,17 @@ class MoveItCpp
   /** \brief Execute a trajectory on the planning group specified by group_name using the trajectory execution manager.
    *  \param [in] group_name MoveIt group to execute for.
    *  \param [in] robot_trajectory Contains trajectory info as well as metadata such as a RobotModel.
-   *  \param [in] blocking If blocking, wait for the trajectory to execute before continuing. Defaults to `true`.
    *  \param [in] controllers An optional list of ros2_controllers to execute with. If none, MoveIt will attempt to find
    * a controller. The exact behavior of finding a controller depends on which MoveItControllerManager plugin is active.
    * \return moveit_controller_manager::ExecutionStatus::SUCCEEDED if successful
    */
   [[deprecated(
       "MoveItCpp::execute() no longer requires a group_name parameter")]] moveit_controller_manager::ExecutionStatus
   execute(const std::string& group_name, const robot_trajectory::RobotTrajectoryPtr& robot_trajectory,
-          bool blocking = true, const std::vector<std::string>& controllers = std::vector<std::string>());
+          const std::vector<std::string>& controllers = std::vector<std::string>());
```

Comment:
This function is deprecated, just remove it

Reply:
To be clear replace the old deprecation of the group_name parameter, to now just three parameters with`bool blocking` as one of them. Then move forward with the true two parameter version of execute.